recipe/meta.yaml

```diff
@@ -11,43 +11,48 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record=files.txt
-  skip: true  # [win or py3k]
+  number: 1
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win]
 
 requirements:
   build:
-    - aipy >=2.1.6
-    - astropy >=1.2
-    - numpy >=1.11
-    - pyephem
-    - python <3
+    - six >=1.10
+    - astropy >=2.0
+    # we require at least numpy 1.14 in pyuvdata
+    - numpy >=1.14
+    - pip
+    - python
     - scipy
     - setuptools
+  host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
Oh, I may have done this wrong in my commit. The requirements that aren't strictly build tools should go in the `host` stanza rather than the `build` stanza — that would be everything except the `compiler()` lines here, I think.